parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -74,8 +74,9 @@ installNvidiaFabricManager() {
 }
 
 installNvidiaContainerToolkit() {
-    MARINER_NVIDIA_CONTAINER_TOOLKIT_VERSION="1.16.2"
-    
+    cat /opt/azure/components.json
+    MARINER_NVIDIA_CONTAINER_TOOLKIT_VERSION=$(jq -r '.Packages[] | select(.name == "nvidia-container-toolkit") | .downloadURIs.azurelinux.current.versionsV2[0].latestVersion' /opt/azure/components.json)
```

Comment:
won't we need a fall-back in case this is executed on an older VHD with an older components.json? I guess that's what you're seeing in failed abe2e's

Reply:
Added the hardcoded fallback option for older VHDs.

Reply:
spoke offline, now have a fall-back for backwards compat